### ISSUE TITLE
Create a CAP Business Service with Node.js Using Visual Studio Code

### ISSUE THREAD

**Issue by FarozanFarman** [sap-tutorials/Tutorials#23124]
Tutorials: https://developers.sap.com/tutorials/cp-apm-nodejs-create-service.html

Issue: Change in Code

At :
STEP 9: 
Test generic handlers:
You can now see the generic handlers shipped with CAP in action.
In the root of your project, create a file called test.http and copy the following requests in it. This file can be used with the [REST client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client) to make requests against your service. The generic handlers CAP provides sent the responses to your requests.
 Check the code for lines for Browse books: 
GET http://localhost:4004/catalog/Books?

This is not the correct URL pointed. And similarly for all other requests.

Here, it should be:
GET http://localhost:4004/odata/v4/catalog/Books?

Please implement the changes accordingly in all the requests.

Labels to be added:
Sprint Label: 2307A
Scenario Label: SCPTest-banode
Landscape Label: trial2
Priority: High

Thanks and Regards,
Farozan

**Comment by renejeglinsky** [comment]
You're totally right, thanks! That has been one of the changes coming the last major release. I'll fix it.
